doc/releases/changelog-0.35.0.md

```diff
@@ -4,25 +4,170 @@
 
 <h3>New features since last release</h3>
 
-<h4>Easy to import circuits 💾</h4>
+<h4>Easy-to-import Qiskit circuits 💾</h4>
 
-* An error message provides clearer instructions for installing PennyLane-Qiskit if the `qml.from_qiskit`
-  function fails because the Qiskit converter is missing.
+* This version of PennyLane makes it easier to import workflows from Qiskit.
   [(#5218)](https://github.com/PennyLaneAI/pennylane/pull/5218)
+  [(#5168)](https://github.com/PennyLaneAI/pennylane/pull/5168)
+
+  The `qml.from_qiskit` function converts a Qiskit
+  [QuantumCircuit](https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.QuantumCircuit) into
+  a PennyLane
+  [quantum function](https://docs.pennylane.ai/en/stable/introduction/circuits.html#quantum-functions).
+  Although `qml.from_qiskit` already exists in PennyLane, we have made a number of improvements to
+  make importing from Qiskit easier. And yes — `qml.from_qiskit` functionality 
+  is compatible with both Qiskit 
+  [1.0](https://docs.quantum.ibm.com/api/qiskit/release-notes/1.0) and earlier 
+  versions! Here's a comprehensive list of the improvements:
+
+  * You can now append PennyLane measurements onto the quantum function returned by
+    `qml.from_qiskit`. Consider this simple Qiskit circuit:
+
+    ```python
+    import pennylane as qml
+    from qiskit import QuantumCircuit
+
+    qc = QuantumCircuit(2)
+    qc.rx(0.785, 0)
+    qc.ry(1.57, 1)
+    ```
+    
+    We can convert it into a PennyLane QNode in just a few lines and then add some `measurements`:
+
+    ```pycon
+    >>> dev = qml.device("default.qubit")
+    >>> measurements = qml.expval(qml.Z(0) @ qml.Z(1))
+    >>> qfunc = qml.from_qiskit(qc, measurements=measurements)
+    >>> qnode = qml.QNode(qfunc, dev)
+    >>> qnode()
+    [tensor(0.00056331, requires_grad=True)]
+    ```
+
+  * Quantum circuits that already contain Qiskit-side measurements can be faithfully converted with
+    `qml.from_qiskit`. Consider this example Qiskit circuit:
+
+    ```python
+    qc = QuantumCircuit(3, 2)  # Teleportation
+  
+    qc.rx(0.9, 0)  # Prepare input state on qubit 0
+  
+    qc.h(1)  # Prepare Bell state on qubits 1 and 2
+    qc.cx(1, 2)
+  
+    qc.cx(0, 1)  # Perform teleportation
+    qc.h(0)
+    qc.measure(0, 0)
+    qc.measure(1, 1)
+  
+    with qc.if_test((1, 1)):  # Perform first conditional
+        qc.x(2)
+    ```
+    
+    This circuit can be converted into PennyLane with the Qiskit measurements still accessible. For example, we can 
+    use those results as inputs to a mid-circuit measurement in PennyLane:
+
+    ```python
+    @qml.qnode(dev)
+    def teleport():
+        m0, m1 = qml.from_qiskit(qc)()
+        qml.cond(m0, qml.Z)(2)
+        return qml.density_matrix(2)
+    ```
+
+    ```pycon
+    >>> teleport()
+    tensor([[0.81080498+0.j, 0.        +0.j],
+        [0.        +0.j, 0.18919502+0.j]], requires_grad=True)
+    ```
```

Comment:
I don't have a strong preference, but I didn't add this initially because it doesn't really add anything to the example.